docs/rules/no-unused-expressions.md

```diff
@@ -39,6 +39,21 @@ a && b()
 a, b()
 
 c = a, b;
+
+a() && function namedFunctionInExpressionContext () {f();}
+
+(function anIncompleteIIFE () {});
+
```

Comment:
:+1:

It took a good night sleep to see how my comment was already obsolete when I wrote it, because you changed the short code example in the introduction not to be a string literal. That helped a lot.

Thank you for testing the emerging guidelines in this situation. Whenever they don’t fit, we will change them!